app/org/sagebionetworks/bridge/models/schedules/ScheduleContext.java

```diff
@@ -176,12 +175,14 @@ public Builder withUserDataGroups(Set<String> userDataGroups) {
             contextBuilder.withUserDataGroups(userDataGroups);
             return this;
         }
-        public Builder withNow(DateTime now) {
-            this.now = now;
+        public Builder withStartsOn(DateTime startsOn) {
+            this.startsOn = startsOn;
             return this;
         }
         public Builder withAccountCreatedOn(DateTime accountCreatedOn) {
-            this.accountCreatedOn = new DateTime(accountCreatedOn, DateTimeZone.UTC);
+            if (accountCreatedOn != null) {
+                this.accountCreatedOn = new DateTime(accountCreatedOn, DateTimeZone.UTC);    
+            }
```

Comment:
Okay, I created a method to test all values being set, and then set through the withContext method, and another method to test none of the values being set, and then that again being copied through the withContext method.